react/modules/sample/src/pages/employee/SampleComponents.js

```
@@ -1213,6 +1216,158 @@ const SampleComponents = () => {
+                },
+              ]} // options to be shown
+              optionsKey={"code"} // key for the options
+            ></MultiSelectDropdown>
+          </div>
+        </LabelFieldPair>
+        {showPopup && (
+          <PopUp
+            className={""}
+            type={"default"} // type of popup
+            heading={"Popup header"} // popup heading
+            children={[
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)